### PR TITLE
Development

### PR DESCRIPTION
**Pull request by wuyepabdul**
In this milestone, I did the following:
- Chose an API to base the application on
- Setup redux store
- Store API data in redux store
- Filter data from redux store and display in CountriesList component
- Write Unit and Integration tests for application
- Write pure function test for application
- Deploy app to Heroku

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hi there, 

Awesome job on making the changes requested by the previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet. Let's work on those.


 ## Required Changes ♻️
_Kindly check the comments under the review._


Happy coding, Cheers!🚀🚀

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it's valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by mikenath223** [comment]
- You had a previous review [here](https://github.com/wuyepabdul/react-capstone/pull/4). Kindly avoid closing PRs with reviews on them. Always ensure you use the same PR link to request a new review.

File: src/components/Country.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+import PropsType from 'prop-types';
+import CovidVirus from './CovidVirus';
+
+const Country = ({ country, handleClick }) => (
+  <div
+    onClick={() => handleClick(country.links[0].href, country.id)}
+    onKeyDown={(e) => e.key === 'Enter' && handleClick(country.links[0].href, country.id)}
+    className="country-div country-div2 country-div3 country-odd-div country-even-div"
+    key={country.id}
+    role="button"
```
Comment (mikenath223) [inline src/components/Country.js:11]: - [x] As requested in the previous review, kindly implement a redux filter/ search functionality on your Homepage so as to enable filtering of countries on the homepage.

Comment (wuyepabdul) [reply src/components/Country.js:11]: Filter function was implemented in the CountriesList component which is the home page.
filter

Comment (wuyepabdul) [reply src/components/Country.js:11]: Please this is my last review. :pray:

**Review by omar-labana — APPROVED** [review]
Hi @wuyepabdul, 

- [x] _[OPTIONAL]_ Kindly update the `Get Started` section in the README to better document running this project locally.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**